src/app/pages/data/manage-data/manage-data.component.html

```diff
@@ -63,10 +63,23 @@ <h4>Records Table</h4>
                            [routerLink]="'/data/projects/' + projId + '/datasets/' + datasetId + '/record/' + flatRecord.id">{{flatRecord.id}}</a>
                     </ng-template>
                 </p-column>
-                <p-column *ngFor="let field of dataset?.data_package?.resources[0]?.schema?.fields"
-                          [field]="field.name" [header]="field.name" [sortable]="true" [style]="getRecordsTableColumnWidth(field.name)">
-                    <ng-template let-flatRecord="rowData" let-col pTemplate="body">
-                        <span [innerHTML]="flatRecord[col.field]"></span>
+                <p-column *ngFor="let field of dataset?.data_package?.resources[0]?.schema?.fields" [field]="field.name"
```

Comment:
For the next time we should probably use the frictionless npm packages to abstract us from the schema implementation details. They are the javascript version of the ones I use server-side:  
https://github.com/frictionlessdata/tableschema-js
https://github.com/frictionlessdata/datapackage-js